server/server-kv.c

```diff
@@ -89,6 +140,9 @@ server_main(int sockfd, char* thread_number)
                 addr_size = sizeof client_addr;
                 clientfd = accept(sockfd, (struct sockaddr *)&client_addr, &addr_size);
                 //sh_print_client_ip(client_addr);
+		#ifdef DEBUG
```

Comment:
Thank you so much for reminding, we will update "#ifdef DEBUG" to  "#ifdef (verbose_print == 0)"